test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1266,17 +1262,19 @@ def expected_postgres_references_manifest(self, model_database=None):
                     'description': 'My table',
                     'docrefs': [
                          {
-                             'documentation_name': 'table_info',
-                             'documentation_package': ''
+                            'documentation_name': 'table_info',
+                            'documentation_package': ''
                          },
                          {
-                             'documentation_name': 'source_info',
-                             'documentation_package': ''
+                            'documentation_name': 'source_info',
+                            'documentation_package': ''
                          }
                      ],
                     'freshness': {},
                     'identifier': 'seed',
-                    'loaded_at_field': None,
+                    # TODO(jeb): Should we create the manifest with
+                    # omit_none=False, so this kind of stuff is always there?
+                    # 'loaded_at_field': None,
```

Comment:
@drewbanin Some fields really feel like they could be None but also are required, and hologram doesn't really let us represent fields like that at the moment. But maybe omitting everywhere None is fine.

Reply:
we'd just pass `omit_none=False` if we wanted to see all those fields.

Reply:
Alternatively, `loaded_at_field` feels like another candidate for that "never omit" field tag I mentioned [here](https://github.com/fishtown-analytics/dbt/pull/1589#discussion_r302615925)

Reply:
Ok - I think creating the manifest with `omit_none=False` is a really good idea!